packages/dfw/src/dfwcomponents/WorkOrderCell/WorkOrderCell.tsx

```diff
@@ -148,19 +148,22 @@ export const WorkOrderCell = ({
                     ))}
                 <Label label={maintenanceType} style={{ marginBottom: 8 }} />
                 {Object.entries(rest).map(([key, value], index) => {
-                    const label = WorkOrderLabelMap[key as keyof WorkOrder];
-                    let displayValue = value;
-                    if (key === "basicStartDate" || key === "basicEndDate") {
-                        displayValue = value ? new Date(value).toLocaleDateString() : "";
+                    if (value) {
```

Comment:
I prefer to exit early by using `if (!value) return null;` to avoid a unnecessary nesting.